lib/restforce/db/synchronizer.rb

```diff
@@ -49,10 +49,11 @@ def run(changes)
       #
       # Returns nothing.
       def update(instance, accumulator)
+        return unless accumulator.changed?(instance.attributes)
```

Comment:
I feel like `#changed?` conveys more information for this system, regarding what has actually happened to the attributes.